app/controllers/places_controller.rb

```diff
@@ -3,6 +3,10 @@ def index
     @places = Place.all
   end
 
+  def show
```

Comment:
This action isn't actually doing anything other than pointing at an <div id='app' /> -- please empty the action (e.g., `def show; end`)